### ISSUE TITLE
help me!  learning rate Suddenly 'Nan' during training

### ISSUE THREAD

**Issue by bill-ysn** [hunglc007/tensorflow-yolov4-tflite#279]
A case where the learning rate suddenly changes to 'Nan' ;
while training your own data set (FISRT_STAGE_EPOCHS) has been changed to 0;
![image](https://user-images.githubusercontent.com/70495121/102166505-3267a780-3ec7-11eb-8c1a-5bfd54863985.png)

**Comment by dinesh-tadi** [comment]
Any fixes found?
Also, I'm getting nan as outputs if I load pre-trained weights (yolov4.weights or yolov4.conv.137). Please suggest any fixes
